application/src/main/resources/ecc.yml

```diff
@@ -32,8 +32,12 @@ connection:
     ## Connection is tried every five second until either the timeout time passes or the connection is successful.
     ##
     timeout:
-      time: 0
+      time: 60
       unit: seconds
+    retryPolicy:
```

Comment:
Perhaps a unit setting for the retryPolicy settings too?

Reply:
Done